public/app/features/alerting/unified/components/contact-points/NotificationTemplates.test.tsx

```diff
@@ -9,9 +11,10 @@ import { AlertmanagerProvider } from '../../state/AlertmanagerContext';
 import { NotificationTemplates } from './NotificationTemplates';
 
 const renderWithProvider = () => {
-  render(
+  return render(
     <AlertmanagerProvider accessType={'notification'}>
       <NotificationTemplates />
+      <AppNotificationList />
```

Comment:
I swear I'd tried this in the past so we could assert on seeing success/failure notifications and couldn't get it to work - nice one though, it's nice to assert like this :+1:

Reply:
💯